src/Dropdown/Dropdown.tsx

```diff
@@ -126,7 +126,7 @@ export const Dropdown = forwardRef(function Dropdown(
           name={name}
           $frontIcon={frontIcon}
           $fallbackStyle={fallbackStyle}
-          value={value ? value : ''}
+          value={value ?? ''}
```

Comment:
nice catch! 
I would suggest to give value a default value as we desctructure it from the props like:
```javascript
value: valueProp = '',
```

and then we can just pass value here
```suggestion
          value={value}
```

Reply:
nice catch as well!:D gave it a default value so now it looks cleaner